site/spark-connect/index.html

```diff
@@ -221,11 +221,11 @@ <h2 id="benefits-of-spark-connect">Benefits of Spark Connect</h2>
 
 <p><strong>Spark Connect workloads are easier to maintain</strong></p>
 
-<p>With the Spark JVM architecture, the client and Spark Driver must run identical software versions.  They need the same Java, Scala, and other dependency versions.  Suppose you develop a Spark project on your local machine, package it as a JAR file, and deploy it in the cloud to run on a production dataset.  You need to build the JAR file on your local machine with the same dependencies used in the cloud.  If you compile the JAR file with Scala 2.13, you must also provision the cluster with a Spark JAR compiled with Scala 2.13.</p>
+<p>With the Spark Classic architecture, the client and Spark Driver must run identical software versions.  They need the same Java, Scala, and other dependency versions.  Suppose you develop a Spark project on your local machine, package it as a JAR file, and deploy it in the cloud to run on a production dataset.  You need to build the JAR file on your local machine with the same dependencies used in the cloud.  If you compile the JAR file with Scala 2.13, you must also provision the cluster with a Spark JAR compiled with Scala 2.13.</p>
```

Comment:
Let's not introduce a new concept, `Spark Classic`. It's not a widely-accepted community concept and this looks ambiguous to me.

Reply:
OK, I removed the reference to Spark Classic.